releasenotes/notes/add-metadata_router-2.0-63829ac5a0528e9d.yaml

```
@@ -0,0 +1,4 @@
+---
+preview:
+  - |
+    Add `MetadataRouter`, a component that routes documents to different edges based on their metadata and specified rules.
```

Matching the docstring
```suggestion
    Add `MetadataRouter`, a component that routes documents to different edges based on the content of their fields
```